packages/sdk/src/swap/services/ApiService.ts

```diff
@@ -52,6 +52,9 @@ const getAssets = async (
   if (chain === Chains.Bitcoin) {
     return [btc$(network, env)];
   }
+  if (chain === Chains.Arbitrum) {
```

Comment:
i think we want to return this for `backspin` only for now

Reply:
also we want to add arbitrum the the return value of `getChains`/`getPossibleDestinationChains`